cfme/fixtures/pytest_selenium.py

```diff
@@ -49,6 +49,24 @@
 class_selector = re.compile(r"^(?:[a-zA-Z][a-zA-Z0-9]*)?(?:[#.][a-zA-Z0-9_-]+)+$")
 
 
+# Monkeypatching WebElement
+_old__repr__ = WebElement.__repr__
+
+
+def __repr__(self):
+    if hasattr(self, "_source_locator"):
+        this, parent = self._source_locator
+        if parent:
+            return "element({}, root={})".format(repr(this), repr(parent))
+        else:
+            return "element({})".format(repr(this))
+    else:
+        return _old__repr__(self)
+
+
+WebElement.__repr__ = __repr__
```

Comment:
Maybe guard this against reloads, something like `if WebElement.__repr__ is not __repr__`?